src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig

```diff
@@ -88,20 +88,6 @@
               <i class="material-icons">help</i>
               <span class="title">{{ 'Help'|trans({}, 'Admin.Global') }}</span>
             </a>
-
-            {% for module in drawerModules %}
```

Comment:
Hi,
it seems that this is a revert of a part from the commit : https://github.com/PrestaShop/PrestaShop/pull/7596
was it wanted ?